README.md

```diff
@@ -13,6 +13,7 @@ github-actions-merger is github actions that merges pull request with commit mes
       "pr_number": ${{ github.event.issue.number }}
       "comment": ${{ github.event.comment.body }}
       "mergers": 'na-ga,0daryo'
+      "enable_auto_merge": false
```

Comment:
We need to add the enable_auto_merge to the inputs.

https://github.com/abema/github-actions-merger/blob/487288a05a33bdf87cadf3be2df074ea2a96e93f/action.yml#L10